src/stream/types.rs

```diff
@@ -34,10 +36,30 @@ impl StreamType {
 impl Apiv2Schema for Url {
 }*/
 
+#[derive(Apiv2Schema, Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct CaptureConfiguration {
```

Comment:
Why is this in stream and not in video ?

Reply:
Hmm, may you tell me, there was a "TODO" comment in the video saying to move it to the stream;

Reply:
Ook so :laughing: